rust/src/index/vector/opq.rs

```diff
@@ -108,7 +110,7 @@ impl OptimizedProductQuantizer {
         let dim = train.num_columns();
 
         // Train few times to get a better rotation matrix. See Faiss.
-        pq.train(&train, metric_type, 4).await?;
+        pq.train(&train, metric_type, 1).await?;
```

Comment:
why the change to only once?

Reply:
Tested a few times , dont see major perf difference at the moment, but speed up OPQ training a bit.